server/app/services/UserService.ts

```diff
@@ -4,13 +4,20 @@ import { UserRepository } from './../repository/UserRepository';
 import { inject, injectable } from 'inversify';
 import TYPES from '../constants/types';
 import bcrypt from 'bcryptjs';
-import { IConfig } from 'config';
+import config, { IConfig } from 'config';
```

Comment:
if you dont refactor my below comment, you can just use this.config in this file instead of importing it